VERSION

```diff
@@ -84,17 +84,17 @@ date="Unreleased developer copy"
 # Version numbers are described in the Libtool current:revision:age
 # format.
 
-libmpi_so_version=30:1:10
+libmpi_so_version=30:1:11
 libmpi_cxx_so_version=30:0:10
-libmpi_mpifh_so_version=31:0:11
-libmpi_usempi_tkr_so_version=30:0:10
-libmpi_usempi_ignore_tkr_so_version=30:0:10
-libmpi_usempif08_so_version=30:0:10
+libmpi_mpifh_so_version=31:0:12
+libmpi_usempi_tkr_so_version=30:0:11
+libmpi_usempi_ignore_tkr_so_version=30:0:11
+libmpi_usempif08_so_version=30:0:11
 
-libopen_rte_so_version=30:1:10
-libopen_pal_so_version=30:1:10
+libopen_rte_so_version=30:1:11
+libopen_pal_so_version=30:1:11
 libmpi_java_so_version=30:0:10
-liboshmem_so_version=30:1:10
+liboshmem_so_version=30:1:11
```

Comment:
Did all these libraries -- Fortran and RTE and PAL and SHMEM -- all actually change since 2.1.1?